index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF">
+  <title>jess codes</title>
+  <link rel="stylesheet" type ="css" href="stylesheets/styles.css">
+</head>
+
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li class="home"><a href ="index.html">jessica weeber</a></li>
+        <li><a href="blog/blog-home.html">blog</a></li>
+        <li><a href="about.html">about me</a></li>
+        <li><a href="portfolio.html">portfolio</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <footer class="index">
```

Comment:
I would name this something more specific, so you know what it is even without the context of the <footer> tag. "footer-index", for example